test/e2e/setup/setup.go

```diff
@@ -140,18 +133,55 @@ func GeneratePasswordForUser(ctx *e2eutil.Context, mdbu mdbv1.MongoDBUser, names
 	return password, e2eutil.TestClient.Create(context.TODO(), &passwordSecret, &e2eutil.CleanupOptions{TestContext: ctx})
 }
 
-func roleDir() string {
-	return envvar.GetEnvOrDefault(roleDirEnv, "/workspace/config/rbac")
-}
+// extractRegistryNameAndVersion splits a full image string and returns the individual components.
+func extractRegistryNameAndVersion(fullStr string) (string, string, string) {
+	splitString := strings.Split(fullStr, "/")
+	registry := strings.Join(splitString[:len(splitString)-1], "/")
 
-func deployDir() string {
-	return envvar.GetEnvOrDefault(deployDirEnv, "/workspace/config/manager")
+	splitString = strings.Split(splitString[len(splitString)-1], ":")
+	version := "latest"
+	if len(splitString) > 1 {
+		version = splitString[len(splitString)-1]
+	}
+	name := splitString[0]
+	return registry, name, version
 }
```

Comment:
This relies on the image being a repo:tag and not with the SHA digest right?
I think this is fine but we could maybe document it in the docstring?

We could even check and maybe return an error, but since this is only in the e2e setup I think we are fine for now!